server/pypi/packages/scipy/patches/chaquopy.patch

```diff
@@ -188,23 +251,34 @@ diff -aur src-orig/scipy/special/_generate_pyx.py src/scipy/special/_generate_py
          for m, codes in enumerate(itertools.product(*all_codes)):
              fused_codes, decs = [], []
              for n, fused_type in enumerate(fused_types):
-diff -aur src-orig/scipy/special/utils/makenpz.py src/scipy/special/utils/makenpz.py
---- src-orig/scipy/special/utils/makenpz.py	2023-09-20 11:08:03.708957161 +0200
-+++ src/scipy/special/utils/makenpz.py	2023-09-21 13:13:09.447909977 +0200
-@@ -6,7 +6,8 @@
+diff --git a/scipy/special/utils/makenpz.py b/scipy/special/utils/makenpz.py
+index f7dd6622a..dc670fe49 100644
+--- a/scipy/special/utils/makenpz.py
++++ b/scipy/special/utils/makenpz.py
+@@ -6,7 +6,8 @@ Build a npz containing all data files in the directory.
  """
  
  import os
 -import numpy as np
 +# Chaquopy: removed `import numpy`: we will "Skip rebuilding if no sources" below, so it's not
 +# needed.
  import argparse
+ from stat import ST_MTIME
  
- from distutils.util import newer  # type: ignore
-diff -aur src-orig/scipy/stats/_generate_pyx.py src/scipy/stats/_generate_pyx.py
---- src-orig/scipy/stats/_generate_pyx.py	2023-09-20 11:08:03.708957161 +0200
-+++ src/scipy/stats/_generate_pyx.py	2023-09-21 13:13:09.463910299 +0200
-@@ -8,6 +8,10 @@
+@@ -47,6 +48,8 @@ def main():
+         outdir_abs = os.path.join(os.getcwd(), args.outdir)
+         outp = os.path.join(outdir_abs, args.dirname + ".npz")
+ 
++    # Chaquopy: Force return
++    return
```

Comment:
This says *what* you're doing, but it doesn't explain *why*. If the rest of the function would cause some problem, then include a comment saying what it was.